### PR TITLE
🌱 Set worker count to 10 everywhere

### PR DESCRIPTION
**Pull request by ncdc**

### CODE REVIEW

**Review by stevekuznetsov — APPROVED** [review]

File: test/e2e/framework/kcp.go [hunk 1]
```diff
@@ -89,7 +89,7 @@ func TestServerWithClientCAFile(clientCAFile string) []string {
 // start a test server with the given token auth file.
 func TestServerArgsWithTokenAuthFile(tokenAuthFile string) []string {
 	return []string{
-		"-v=4",
```
Comment (sttts) [inline test/e2e/framework/kcp.go:92]: this is undesired, isn't it?

Comment (ncdc) [reply test/e2e/framework/kcp.go:92]: It's for debugging the failures

Comment (sttts) [reply test/e2e/framework/kcp.go:92]: for debugging sure, but to merge? We already went from 2 to 4 sometime ago. Now 5?

Comment (ncdc) [reply test/e2e/framework/kcp.go:92]: Not to merge, no

**Comment by stevekuznetsov** [comment]
@ncdc I assume we still want this?

**Comment by ncdc** [comment]
Replaced by #2960 
/close

**Pull request closed without merging**